client/my-sites/importer/importing-pane.jsx

```diff
@@ -141,10 +225,11 @@ export default React.createClass( {
 					/>
 				}
 				{ this.isImporting() && (
-					percentComplete
+					percentComplete >= 0
```

Comment:
mentioned elsewhere, but I think you're disallowing 0% complete from being displayed within your `hasProgressInfo()` check.